docs/reference/commandline/node_inspect.md

```diff
@@ -1,9 +1,15 @@
 ---
-title: "node inspect"
-description: "The node inspect command description and usage"
-keywords: ["node, inspect"]
+description: The node inspect command description and usage
+keywords:
+- node, inspect
+menu:
+  main:
+    parent: smn_cli
+title: node inspect
 ---
 
+**Warning:** this command is part of the Swarm management feature introduced in Docker 1.12, and might be subject to non backward-compatible changes.
```

Comment:
Bit confused by this message; what's the backward-incompatibility (as this command didn't exist before); maybe this originally referred to it being an `RC` / preview, so changes could still occur?